scene/gui/graph_node.cpp

```diff
@@ -392,7 +392,7 @@ void GraphNode::_notification(int p_what) {
 				}
 
 				// Draw slot stylebox.
-				if (s.draw_stylebox) {
+				if (s.draw_stylebox && get_child_count() != 0) {
```

Comment:
suggestion, change line 370 instead
```c++
if (E.key < 0 || E.key >= cache_y.size() || get_child_count() == 0) {
	continue;
}
```
**OR** add an if-clause before that if-clause in line 370:
```c++
if (get_child_count() == 0) {
	break;
}
```

Reason is: `slot_info` and `cache_y` are outdated / dirty, and not yet recalculated

**EDIT**: if it's desirable to draw slots at least once, even if there are no children, then the 2nd proposal with the `break` can be added in line 393 instead

Reply:
Thanks for the feedback! I'll move the check to line 370 as per the first proposal. That was my original idea but I wasn't sure how that might affect other things dependent on the code between 370 and 395. Seems like it's no problem!